### ISSUE TITLE
Erroneous: You can't use nested r.row

### ISSUE THREAD

**Issue by niieani** [neumino/rethinkdbdash#79]
The same query works perfectly using the official JS client, yet I get an error when using ```rethinkdbdash```, stating that I can't use a nested ```r.row```. Here's my query:

```javascript
r.table(test)
  .get(id).merge(
    r.object(
      'components', r.row('components')
      .eqJoin(function(t){return t}, r.table(test))
      .zip()
    ));
```

Best,
Bazyli

**Comment by neumino** [comment]
Fix released in 1.16.14.
Thanks for the report @niieani!

**Comment by niieani** [comment]
Your welcome, and thanks for the quick fix too, @neumino! :)
